src/state/connection-mixin.ts

```diff
@@ -145,6 +145,11 @@ export const connectionMixin = <T extends Constructor<HassBaseEl>>(
         }
       });
 
+      // If they reconnect during startup we need to reload
+      // when startup is completed
+      conn.subscribeEvents(() => {
+        location.reload();
+      }, "homeassistant_start");
```

Comment:
This is too aggressive. Most pages will actually do just fine. It's just generated Lovelace UIs opened while starting up that have issues.

Reply:
We also shouldn't add non-data logic in the `state` mix-ins as they are used for other projects too (ie cast). We have logic like this in `home-assistant.ts` for when we detect a version upgrade (And a reload is needed)

Reply:
Agreed.  Probably makes sense to close this PR and go the route @bramkragten suggested above in https://github.com/home-assistant/frontend/pull/6042#issuecomment-633624026 

I'll leave this open and marked as a draft for now though so there is a place to comment.